cjworkbench/tests/i18n/test_trans.py

```diff
@@ -56,6 +60,46 @@ def test_no_html_escape(self):
             'Hello <a href="/you?a=n&b=e">you > > there</a> my & friend',
         )
 
+    # Tests that plurals, selects and nested messages are fully supported
+    def test_icu_support(self):
+        message = (
+            "Hello {a}, you have {g, select,"
+            "   male {{n, plural,"
+            "       =0 {no boys} one {# little boy} other {# little boys}"
```

Comment:
Could we please nix the word "little" from this test? It triggers me. (If I heard a random man on the street say something about "little boys," I'd suspect he's a pervert.)